webboot/webboot.go

```diff
@@ -26,43 +33,134 @@ var (
 	ipv4      = flag.Bool("ipv4", true, "use IPV4")
 	ipv6      = flag.Bool("ipv6", true, "use IPV6")
 	osSystems = map[string]*webboot.Distro{
-		"tinycore": &webboot.Distro{"/boot/vmlinuz64", "/boot/corepure64.gz", "console=tty1", "https:louis.com"},
-		"TestISO":  &webboot.Distro{"/kernel", "/initiso.cpi", "console=ttyS0 console=tty0", "https:louis.com"},
+		"tinycore": &webboot.Distro{"/boot/vmlinuz64", "/boot/corepure64.gz", "console=tty1", "http://tinycorelinux.net/10.x/x86_64/release/CorePure64-10.1.iso"},
+		"TestISO":  &webboot.Distro{"/boot/kernel", "/boot/initiso.cpi", "console=ttyS0 console=tty0", "http://releases.ubuntu.com/18.04.2/ubuntu-18.04.2-desktop-amd64.iso"},
 	}
 )
 
+// parseArgs takes the arg user passes e.g. name of OS or link
+// if arg matches the name of one of the supported OSses, parseArgs retrieves its link from the map & assigns filename to be arg.
+// The steady filename allows it to be consistent with the key in the supported OSses map as well as the one the user described the OS as
```

Comment:
I mean to say that the filename will remain identical to the name of the OS the user passed in. It is more convienent for the user and for us to refer to it as the same name we have saved up in the bookmark and since the user passed it in as 'TinyCore' and the end result of the filename is 'TinyCore' it leads to less confusion for both the users and the programmers

Reply:
But it can lead into confusion if I word it like that, and also, now that I think of it. Changing the comments to be the 3 line comments you suggested above, I do not believe I even need a comment along these lines because it does not add much value nor is it necessary. The 3 line comments you suggest above provide a more meaningful message of the function. I will omit this piece of comment